docs/blog/2021-community-update.rst

```diff
@@ -0,0 +1,122 @@
+and moderation is a necessary part of that work.
+
+We have also just launched a new `organizational structure and governance WEP`_,
+which will help to shape the structure of how Write the Docs operates.
+This WEP will define all the various teams in the community,
+as well as how decisions are made at the organizational level.
+The proposal adds the concept of a Community Council,
+which will be made up of the team leads of all the community teams.
+The Community Council will make many decisions around the community,
+allowing more people to share in the official leadership structure.
+
+We hope that more people in our community will participate in the WEP process,
+so please read through the proposal and give us feedback.
+
+.. _Slack Moderation WEP: https://github.com/writethedocs/weps/blob/master/accepted/WEP0002.rst#wep2---slack-community-moderation-procedures
+.. _organizational structure and governance WEP: https://github.com/writethedocs/weps/pull/5
+
+Write the Docs store
+--------------------
+
+We've heard from some of the folks in the community that they are missing all the swag we used to give away at our conferences.
+We always had lots of little goodies to give folks across the years, including stickers, notebooks, shirts, hoodies, and other bigger items in some of our raffle giveaways.
+We're excited to be able to bring back a small version of that for those of you who want to wear your documentarian pride on your sleeves :)
+
+The `Write the Docs Store`_ is now live with lots of good things. **We have stickers, mugs, phone cases, bags, prints, and notebooks** with these designs:
+
+* `Docs or it didn't happen`_ swag in dark & light
+* `Sketchnotes`_ from all the talks at our Portland 2021 conference
+* `Community logos`_ which provide a simple way to show your WTD pride
+
+We have tried to find a vendor which has reasonable shipping costs so that we can keep the costs down for our community, but it can still add up.
+We recommend trying to do a group purchase with a meetup or company so that you can reduce the costs involved, and get a few different options of the various items.
+If you also want a bulk order of stickers or similar swag for a meetup, we can also directly ship them to you for free if you email us at support@writethedocs.org.
+
+.. _Write the Docs Store: https://writethedocs.threadless.com/
+.. _Docs or it didn't happen: https://writethedocs.threadless.com/collections/docs-or-it-didnt-happen
+.. _Sketchnotes: https://writethedocs.threadless.com/collections/wtd-portland-2021-sketchnotes
+.. _Community logos: https://writethedocs.threadless.com/collections/community-logos
+
+Job Board & Newsletter sponsorship pricing updates
+--------------------------------------------------
+
+We have been seeing great success with our `job board`_ and `newsletter sponsorships`_.
+The job listings have been getting a bit overwhelming,
+so we've decided to try and mitigate that by increasing the prices.
```

Comment:
This sounds a bit negative, I'm thinking something like:

```
We're glad to see that these are so popular and have
decided to bring the pricing closer to the value they provide.
```

We're raising them because things are good - not because something bad is happening that we need to fix. Or at least, that's what I think it should look like.